{
  "Header": {
    "Original": "CVE-2007-3919 ((1) xenbaked and (2) xenmon.py in Xen 3.1 and earlier allow local user ...)",
    "Line": 561128,
    "ID": "CVE-2007-3919",
    "Description": "((1) xenbaked and (2) xenmon.py in Xen 3.1 and earlier allow local user ...)"
  },
  "Annotations": [
    {
      "Original": "{DSA-1395-1}",
      "Line": 561129,
      "Type": "xref",
      "Bugs": [
        "DSA-1395-1"
      ]
    },
    {
      "Original": "- xen-unstable 3.0-unstable+hg11561-1 (low; bug #464044)",
      "Line": 561130,
      "Type": "package",
      "Package": "xen-unstable",
      "Kind": "fixed",
      "Version": "3.0-unstable+hg11561-1",
      "Severity": "low",
      "BugNo": 464044
    },
    {
      "Original": "- xen-3 3.1.2-1 (low)",
      "Line": 561131,
      "Type": "package",
      "Package": "xen-3",
      "Kind": "fixed",
      "Version": "3.1.2-1",
      "Severity": "low"
    }
  ]
}